### ISSUE TITLE
Remove documentation about package install ?

### ISSUE THREAD

**Issue by jooola** [libretime/libretime#1879]
The libretime debian package repository is out of date with the current libretime version.

Should we remove the documentation about it until we upgrade the package repository ?

https://libretime.org/docs/admin-manual/setup/install/#using-packages

@paddatrapper

**Comment by paddatrapper** [comment]
Yeah I think it is a good idea to remove it. There is a fair amount of work to get the next version packaged, which I haven't had a chance to do yet
